exercises/twelve-days/twelve_days.exs

```diff
@@ -0,0 +1,61 @@
+defmodule TwelveDays do
+  @doc """
+  Given a `number`, return the word representation of it, or "a" for one.
+
+  1 => "a",
+  2 => "two",
+  3 => "three",
+  etc...
+  """
+  @spec count(number :: integer) :: String.t()
+  def count(number) do
```

Comment:
This feels to me like it's too specifically guiding folks towards an implementation. I'd prefer something a little more rough grained here.